src/main/java/org/gridsuite/modification/server/repositories/NetworkModificationRepository.java

```diff
@@ -100,9 +100,10 @@ public void saveModifications(UUID groupUuid, List<ModificationEntity> modificat
                 .collect(Collectors.toList());
     }
 
+    @Transactional
     public List<ModificationInfos> getModifications(UUID groupUuid) {
         ModificationGroupEntity group = getModificationGroup(groupUuid);
-        return this.modificationRepository.findAllBaseByGroupId(group.getId())
+        return this.modificationRepository.findAllByGroupId(group.getId())
```

Comment:
So, you can remove the no more used query findAllBaseByGroupId in ModificationRepository class

Reply:
I need these data or else I'll ask for se same data in another request